inference-engine/tests/functional/plugin/myriad/ngraph/conversions/dynamic_shape_resolver.cpp

```diff
@@ -41,7 +42,9 @@ class DynamicShapeResolverTests : public CommonTestUtils::TestsCommon {
 
 private:
     void triggerConversionToCNNNetwork() {
-        cnnNetwork.begin();
+        auto & inetwork = (const InferenceEngine::ICNNNetwork &)cnnNetwork;
+        InferenceEngine::details::CNNNetworkIterator i(&inetwork);
+        (void)i;
```

Comment:
Could you please explain what are trying to do here? Code looks really strange..

Reply:
@gladilov-gleb do you see the name of the function? `triggerConversionToCNNNetwork` looks very descriptive :)

Reply:
re-wrote with other method which triggers implicit conversion. Hope it looks more friendly for you now :)